coding/python_testing.rst

```diff
@@ -4,6 +4,156 @@ Python Unit Testing
 
 This page provides technical guidance to developers writing unit tests for DM's Python code base.
 See :doc:`unit_test_policy` for an overview of LSST Stack testing.
+If you have legacy DM :mod:`unittest` ``suite``-based code please refer to tech note `SQR-012`_ for porting instructions.
```

Comment:
Yes, I was trying not to spend time documenting a scheme that we had inherited through cargo-culting rather than documented policy. I'll have a go at putting enough words in to help people spot the old style without documenting the old style here (SQR-012 does describe the old and the new).